### PR TITLE
Require static imports for immutable collection collectors

### PR DESCRIPTION
**Pull request by kokosing**
Require static imports for immutable collection collectors

### CODE REVIEW

File: airbase-policy/src/main/resources/checkstyle/airbase-checks.xml [hunk 1]
```diff
@@ -70,6 +70,18 @@
         <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*Math\.toIntExact" />
         <property name="message" value="Math.toIntExact should only be used with static imports" />
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*ImmutableMap\.toImmutableMap" />
```
Comment (findepi) [inline airbase-policy/src/main/resources/checkstyle/airbase-checks.xml:74]: I know this is copied from above, but wonder if 
```
^(?!import).* ImmutableMap\.toImmutableMap
```
would be enough.

**Review by findepi — APPROVED** [review]

File: airbase-policy/src/main/resources/checkstyle/airbase-checks.xml (hunk 1 above)
Comment (kokosing) [reply airbase-policy/src/main/resources/checkstyle/airbase-checks.xml:74]: I don't know. I thought that this way is for a reason.

Comment (electrum) [reply airbase-policy/src/main/resources/checkstyle/airbase-checks.xml:74]: I believe that will work, but we can clean that up separately

Comment (electrum) [reply airbase-policy/src/main/resources/checkstyle/airbase-checks.xml:74]: Filed https://github.com/airlift/airbase/issues/241 for that cleanup

**Pull request merged**